internal/scanner/scanner.go

```diff
@@ -89,11 +89,16 @@ func (s *Scanner) ScanNamespace(nsName string) error {
 	// create PolicyReport
 	namespacedsReport := report.NewPolicyReport(namespace)
 	namespacedsReport.Summary.Skip = skippedNum
+	// old policy report to be used as cache
+	previousNamespacedReport, err := s.reportStore.GetPolicyReport(nsName)
+	if err != nil {
+		log.Error().Err(err).Msg("error getting PolicyReport from store")
```

Comment:
Testing it locally, with an existing PolicyReport, and it always errors here, hence it never uses the previous report.